specification/monitor/resource-manager/microsoft.insights/2017-05-01-preview/diagnosticsSettingsCategories_API.json

```diff
@@ -0,0 +1,266 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MonitorManagementClient",
+    "version": "2017-05-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{resourceUri}/providers/microsoft.insights/diagnosticSettingsCategories/{name}": {
+      "get": {
+        "tags": [
+          "DiagnosticSettingsCategories"
+        ],
+        "operationId": "DiagnosticSettingsCategory_Get",
+        "description": "Gets the diagnostic settings category for the specified resource.",
```

Comment:
Please indicate some of the typical values resourceUri could take to help customers understand this.

Reply:
This is explicit in the example with a logic app.

Reply:
Is it too difficult to add it to the description? SDK users rely on description, API docs rely on description.

Reply:
Any change in the description breaks the generation, so it requires more work that touches all our classes. I created a task on us to fix that.